servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ClientClosureRaceTest.java

```diff
@@ -175,13 +176,13 @@ private void runIterations(Callable<Single<?>> test) throws Exception {
     private SingleAddressHttpClientBuilder<HostAndPort, InetSocketAddress> newClientBuilder() {
         final RetryingHttpRequesterFilter.Builder retryBuilder = new RetryingHttpRequesterFilter.Builder();
         return HttpClients.forSingleAddress(HostAndPort.of((InetSocketAddress) serverSocket.getLocalSocketAddress()))
-                .appendClientFilter(retryBuilder.maxRetries(Integer.MAX_VALUE)
-                        .retryFor((md, t) ->
+                .appendClientFilter(retryBuilder
+                        .retryOther((md, t) ->
                             // This test has the server intentionally hard-close the connection after responding
                             // to the first request, however some tests use pipelining and may write multiple requests
                             // on the same connection which would result in a non-retryable exception. Since this test
                             // doesn't care about idempotency it should always retry.
-                            true
-                        ).buildWithConstantBackoffFullJitter(ofNanos(1)));
+                           ofConstantBackoffFullJitter(ofNanos(1), MAX_VALUE)
```

Comment:
The snapshot build is currently failing with: https://github.com/apple/servicetalk/runs/4476095925
```
Check failure on line 184 in servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ClientClosureRaceTest.java 
Comment has incorrect indentation level 28, expected is 27, indentation should be the same level as line 185.
```